packages/l/libpillowfight/monitoring.yml

```diff
@@ -0,0 +1,6 @@
+releases:
+  id: 26408
+  rss: https://pypi.org/pypi/pypillowfight/json
```

Comment:
This is not a valid rss feed. It is usually ended with `rss` or `atom`. If you cannot find any rdd feed you can just fill it as null.
```
rss: ~
```

Reply:
Oops, I copied the wrong one. Fixed